embedding_pdf_documents/ray_cluster_setup.md

```diff
@@ -0,0 +1,87 @@
+# Creating a Ray cluster with 20 GPUs
```

Comment:
This file should be renamed README.md so that when people open this directory, they see the instructions correctly on github.

Reply:
agreed